util/config/config.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2023 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package config
```

Comment:
Not sure this is a descriptive package name. Maybe `kubeclient` since it's related to client-go functionality?

Reply:
Renamed to `kubeclient`